package.json

```diff
@@ -84,27 +85,30 @@
     "prettier": "2.8.8",
     "puppeteer": "10.0.0",
     "quicktype-core": "23.0.71",
-    "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react": "^16.13.1",
     "rimraf": "5.0.1",
     "rollup": "3.26.0",
     "semver": "7.5.4",
     "shell-quote": "1.8.1",
     "stencil-eslint-core": "0.3.1",
-    "storybook": "6.5.16",
     "storybook-addon-themes": "6.1.0",
     "storybook-rtl-addon": "0.3.3",
-    "stylelint": "15.10.3",
+    "storybook": "6.5.16",
+    "style-dictionary": "3.7.1",
     "stylelint-config-recommended-scss": "12.0.0",
     "stylelint-use-logical-spec": "5.0.0",
+    "stylelint": "15.10.3",
     "tailwindcss": "3.3.3",
+    "token-transformer": "0.0.27",
     "ts-jest": "27.1.5",
     "ts-node": "10.9.1",
     "turbo": "1.10.13",
     "type-fest": "3.11.1",
     "typescript": "4.9.5",
     "updtr": "4.0.0",
-    "workbox-build": "7.0.0"
+    "workbox-build": "7.0.0",
+    "yargs": "17.7.2"
```

Comment:
Why do we need yargs?

Reply:
I couldn't find any usages of yargs with a quick grep, so can you uninstall `yargs` and `@types/yargs`?

Reply:
removing for now. When we move token-transformer to it's own package we'll probably want yargs to do the CLi interface